index.yaml

```diff
@@ -1,14 +1,52 @@
 apiVersion: v1
 entries:
   quickwit:
+  - apiVersion: v2
+    appVersion: edge
+    created: "2022-11-05T14:43:45.815105+01:00"
+    description: Open-Source & Cloud-Native Log Management at any scale.
+    digest: 105ab64b2c362c26ed8c45d06e3aabdaa30b76d83784f374c0c4c5d7b4d612be
+    home: https://quickwit.io/
+    icon: https://avatars.githubusercontent.com/u/98504233?s=200&v=4
+    keywords:
+    - quickwit
+    maintainers:
+    - email: mkhpalm@gmail.com
+      name: mkhpalm
+    name: quickwit
+    sources:
+    - https://github.com/quickwit-oss/quickwit/
+    type: application
+    urls:
+    - https://helm.quickwit.io/quickwit-0.1.6.tgz
+    version: 0.1.6
+  - apiVersion: v2
+    appVersion: edge
+    created: "2022-11-05T14:43:45.814491+01:00"
+    description: Open-Source & Cloud-Native Log Management at any scale.
+    digest: 248afa1f96fe639008bf71c25346d89e812e10d4b7c0b8d2e720f999bf51e25f
+    home: https://quickwit.io/
+    icon: https://avatars.githubusercontent.com/u/98504233?s=200&v=4
+    keywords:
+    - quickwit
+    maintainers:
+    - email: mkhpalm@gmail.com
+      name: mkhpalm
+    name: quickwit
+    sources:
+    - https://github.com/quickwit-oss/quickwit/
+    type: application
+    urls:
+    - https://helm.quickwit.io/quickwit-0.1.5.tgz
+    version: 0.1.5
   - apiVersion: v2
     appVersion: 1.16.0
-    created: "2022-06-25T17:03:47.065492189-04:00"
+    created: "2022-11-05T14:43:45.813986+01:00"
     description: A Helm chart for Kubernetes
     digest: b375d2a990c0d8418cb94f71e959ace57851d11953a1b02b791fd8f04e69b047
     name: quickwit
     type: application
     urls:
-    - https://github.com/quickwit-oss/helm-charts/quickwit-0.1.0.tgz
+    - https://helm.quickwit.io/quickwit-0.1.0.tgz
     version: 0.1.0
```

Comment:
`0.1.6` here too?

Reply:
Ok, my bad, wrong comment, nothing to change here.

Reply:
the 0.1.6 comes before in the file. You have one item int he list per chart version, which targets its own `.tgz` file. 
this is generated by : 
`helm repo index --url https://helm.quickwit.io/ . `